src/UnifiedFFI-Tests/FFIExternalStructureFieldParserTests.class.st

```diff
@@ -30,13 +30,14 @@ FFIExternalStructureFieldParserTests >> testParseFieldsStructure [
 		structure: FFITestStructure.
 		
 	self assert: fieldSpec notNil. 
-	self assert: fieldSpec fieldNames equals: #(byte short long float double int64 ulong).
+	self assert: fieldSpec fieldNames equals: #(byte short long float double int64 ulong size).
```

Comment:
why not size_t to follow the same naming than C/C++?

Reply:
Changed to have the name match the type.